website/docs/r/org_vdc_template_instance.html.markdown

```diff
@@ -0,0 +1,143 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_org_vdc_template_instance"
+sidebar_current: "docs-vcd-resource-org-vdc-template-instance"
+description: |-
+  Provides a resource to instantiate VDCs from a VDC Template in VMware Cloud Director.
+---
+
+# vcd\_org\_vdc\_template\_instance
+
+Provides a resource to instantiate VDCs from a [VDC Template](/providers/vmware/vcd/latest/docs/resources/org_vdc_template) in VMware Cloud Director.
+Supported in provider *v3.13+*
+
+## Example Usage
+
+```hcl
+data "vcd_org" "org" {
+  name = "my_org"
+}
+
+data "vcd_provider_vdc" "pvdc1" {
+  name = "nsxTPvdc1"
+}
+
+data "vcd_provider_vdc" "pvdc2" {
+  name = "nsxTPvdc2"
+}
+
+data "vcd_external_network_v2" "ext_net" {
+  name = "nsxt-extnet"
+}
+
+data "vcd_network_pool" "np1" {
+  name = "NSX-T Overlay 1"
+}
+
+resource "vcd_org_vdc_template" "tmpl" {
+  name               = "myTemplate"
+  tenant_name        = "myAwesomeTemplate"
+  description        = "Requires System privileges"
+  tenant_description = "Any tenant can use this"
+  allocation_model   = "AllocationVApp"
+
+  compute_configuration {
+    cpu_limit         = 0
+    cpu_guaranteed    = 20
+    cpu_speed         = 256
+    memory_limit      = 1024
+    memory_guaranteed = 30
+  }
+
+  provider_vdc {
+    id                  = data.vcd_provider_vdc.pvdc1.id
+    external_network_id = data.vcd_external_network_v2.ext_net.id
+  }
+
+  provider_vdc {
+    id                  = data.vcd_provider_vdc.pvdc2.id
+    external_network_id = data.vcd_external_network_v2.ext_net.id
+  }
+
+  storage_profile {
+    name    = "*"
+    default = true
+    limit   = 1024
+  }
+
+  network_pool_id = data.vcd_network_pool.np1.id
+
+  readable_by_org_ids = [
+    data.vcd_org.org.id
+  ]
+}
+
+resource "vcd_org_vdc_template_instance" "my_instance" {
+  org_vdc_template_id = vcd_org_vdc_template.tmpl.id
+  name                = "myInstantiatedVdc"
+  description         = "A new VDC"
+  org_id              = data.vcd_org.org.id
+
+  delete_instantiated_vdc_on_removal = true
```

Comment:
I think we should have false for this in the example, given that someone may blindly copy it. And maybe add a comment.

Reply:
Changed + Added comment